### PR TITLE
minor fixes to tests

### PR DESCRIPTION
**Pull request by irimiecernea**
Please review.

### CODE REVIEW

File: lola-tech/test/hooks/hooks.js [hunk 1]
```diff
@@ -4,13 +4,17 @@ let assert = require("chai").assert;
 
 class Hooks extends Page {
   getWebElement(elementName, page) {
+    console.log(elementName);
     elementName = Utils.formatLocator(elementName);
+    console.log(elementName);
```
Comment (radurusu) [inline lola-tech/test/hooks/hooks.js]: Please remove the console logs

**Review by radurusu — CHANGES_REQUESTED** [review]
Please make the changes

File: lola-tech/test/hooks/hooks.js (hunk 1 above)
Comment (irimiecernea) [reply lola-tech/test/hooks/hooks.js]: I made the requested changes. Please review again.

**Review by radurusu — APPROVED** [review]
Looks good

**Pull request merged**